Dockerfile

```diff
@@ -1,30 +1,56 @@
+### Build stage for the website frontend 
 FROM node:10 as website
+
 WORKDIR /code
+
 COPY ./website/package.json ./
 COPY ./website/package-lock.json ./
+
+# install dependency
 RUN npm install
+
 COPY ./website/ ./
+
 RUN npm run build
 
+### Build stage for the website backend server
 FROM golang:1.13.8 as server
+
 WORKDIR /code
+
+# Environment variable
 ENV GOOS=linux
 ENV GARCH=amd64
 ENV CGO_ENABLED=0
 ENV GO111MODULE=on
+
 COPY ./go.mod ./
 COPY ./go.sum ./
+
 RUN go mod download
+
 COPY ./proto/ ./proto/
 COPY ./main.go ./main.go
 COPY ./internal/ ./internal
+
 RUN go build -o server
 
+### Server 
 FROM alpine:3.10
-RUN apk add iptables
-RUN apk add wireguard-tools
+
+# Environment variable
```

Comment:
Hey, it'd be great to move the `apk add` commands back to the top under the `FROM` statement.

This helps us avoid rebuilding those expensive layers of the container when we need to update the ENV variables. Better caching, better download speed for users :)